Sources/SwiftParser/Expressions.swift

```diff
@@ -512,13 +512,19 @@ extension Parser {
   }
 
   @_spi(RawSyntax)
-  public mutating func parseDottedExpressionSuffix() -> (
-      period: RawTokenSyntax, name: RawTokenSyntax,
-      declNameArgs: RawDeclNameArgumentsSyntax?,
-      generics: RawGenericArgumentClauseSyntax?
+  public mutating func parseDottedExpressionSuffix<R: RawSyntaxNodeProtocol>(leading: R?) -> (
+    unexpectedPeriod: RawUnexpectedNodesSyntax?,
+    period: RawTokenSyntax,
+    name: RawTokenSyntax,
+    declNameArgs: RawDeclNameArgumentsSyntax?,
+    generics: RawGenericArgumentClauseSyntax?
   ) {
-    assert(self.at(any: [.period, .prefixPeriod]))
-    let period = self.consumeAnyToken(remapping: .period)
+    assert(self.at(.period))
+    let (unexpectedPeriod, period) = self.consumeMemberPeriod(leading: leading)
+    if currentToken.isAtStartOfLine {
```

Comment:
As of Swift 5.7,
```
foo  .
bar
```
i.e `"foo .\nbar"` is a valid member access syntax. I think this changes the behavior.

Reply:
Ah, whoops. Thanks. Guess I'll check for the previous node's trivia here as well :(